Gemfile.lock

```diff
@@ -14,34 +13,31 @@ GEM
       execjs
     coffee-script-source (1.10.0)
     colorator (0.1)
-    ethon (0.8.1)
+    ethon (0.9.0)
```

Comment:
Would rather comment on the whole file, but it seems like it would be a good idea to keep Gemfile changes out of feature-specific changes. No argument against upgrading dependencies, but that's not related to the rest of the PR at all.